basic/exercise2/script2.js

```diff
@@ -10,6 +10,15 @@ function validateSide(side) {
 };
 
 function triangleArea(side1, side2, side3) {
+    if (side1 > side2 + side3) {
```

Comment:
You have a repetitive code - plese create a function that encapsulates it and call it three times here.